.github/BECOMING_A_CORE_CONTRIBUTOR.md

```diff
@@ -17,53 +18,48 @@ With that said, the Lightning team will be diverse and a reflection of an inclus
 
 ## Responsibilities:
 
-The responsibilities mainly revolve around 3 things.
+Here, we describe general expectations from core contributors:
 
 ### Github issues
 
-- Here we want to help users have an amazing experience. These range from questions from new people getting into DL to questions from researchers about doing something esoteric with Lightning
-  Often, these issues require some sort of bug fix, document clarification or new functionality to be scoped out.
+- Help users so they have an amazing experience. Issues range from questions from new people getting into DL to questions from researchers about doing something esoteric with Lightning.
+  They often require some sort of bug fix, document clarification, or new functionality to be scoped out. You can help them solve their issues and guide them to completion.
 
-- To become a core member you must resolve at least 10 Github issues which align with the API design goals for Lightning. By the end of these 10 issues I should feel comfortable in the way you answer user questions
-  Pleasant/helpful tone.
+- Weigh in on discussions in a timely fashion. Most importantly, on the RFCs (request for comments) that will shape the future of Lightning.
+  There are some big decisions which the project must make. For these, we expect core contributors to have something meaningful to add, especially if it’s their area of expertise.
```

Comment:
```suggestion
  There are some big decisions which the project must make. For these, we expect core contributors to have something meaningful to add, especially if it’s their area of expertise. 
```

Reply:
Mmm what did you intend to suggest? I don't see any changes other than an added space after the period